webskins/_default_/pub/selectize-0.12.1-dark.css

```diff
@@ -0,0 +1,32 @@
+:root {
```

Comment:
Please add a comment that it's ZNC's custom code, and not upstream.

Renaming this file to e.g. selectize-znc-dark.css would also help

Reply:
Done.